src/glossary.rst

```diff
@@ -3627,6 +3697,14 @@ An :dt:`operator expression` is an :t:`expression` that involves an operator.
 :dp:`fls_qdszbyeuo7w1`
 See :s:`OperatorExpression`.
 
+.. _fls_C5DiCsvsaBsj:
+
+opt-out trait bound
+^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_wS4EzN0N1GDP`
+An :dt:`opt-out trait bound` is ??? HELP LUKAS!
```

Comment:
An opt-out trait bound is a `?Sized` thing (the question mark bounds).
So something like:
> An opt-out trait bound is a trait bound that allows opting out of implicity introduced trait bounds.

Reply:
Now defined.